tests/acceptance/features/apiSharingNgLinkShare/linkShare.feature

```diff
@@ -2300,3 +2300,374 @@ Feature: Create a share link for a resource
       """
     And the public should be able to download file "textfile.txt" from the last link share with password "%public%" and the content should be "to share"
     And the public download of file "textfile.txt" from the last link share with password "$heLlo*1234*" should fail with HTTP status code "401" using shareNg
+
+  @issue-7879
+  Scenario Outline: try to create a link share of a Personal and Share drive using permission endpoint
+    When user "Alice" tries to create the following space link share using permission endpoint of the Graph API:
+      | space           | <drive>            |
+      | permissionsRole | <permissions-role> |
+      | password        | %public%           |
+    Then the HTTP status code should be "400"
+    And the JSON data of the response should match
+      """
+      {
+        "type": "object",
+        "required": ["error"],
+        "properties": {
+          "error": {
+            "type": "object",
+            "required": [
+              "code",
+              "innererror",
+              "message"
+            ],
+            "properties": {
+              "code": {
+                "const": "invalidRequest"
+              },
+              "innererror": {
+                "type": "object",
+                "required": [
+                  "date",
+                  "request-id"
+                ]
+              },
+              "message": {
+                "const": "<message>"
+              }
+            }
+          }
+        }
+      }
+      """
+    Examples:
+      | permissions-role | drive    | message                                   |
+      | view             | Shares   | no share permission                       |
+      | edit             | Shares   | no share permission                       |
+      | upload           | Shares   | no share permission                       |
+      | createOnly       | Shares   | no share permission                       |
+#      | blocksDownload   | Shares   | invalid link type                         |
```

Comment:
```suggestion
      | blocksDownload   | Shares   | invalid link type                         |
```

Reply:
currently block download doesn't work

Reply:
yes, but we don't comment out the scenario. we put them in expected-to fail list